src/web/components/AccountComponent.html

```diff
@@ -67,13 +73,36 @@
                     this.set({ errorPassword: error })
                 }.bind(this))
             },
-            t411Token: function () {
+            updateT411Profile: function () {
+                window.app.api.get('/t411/profile/refresh').then(function (response) {
```

Comment:
Where do you want me to place the return here ?
`return window.app.api.get()..` ?